web/src/components/molecules/Asset/UploadModal/tab1.tsx

```diff
@@ -0,0 +1,27 @@
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Upload, { UploadProps } from "@reearth-cms/components/atoms/Upload";
+
+const { Dragger } = Upload;
+
+type Props = {
+  uploadProps: UploadProps;
+};
+
+const Tab1: React.FC<Props> = ({ uploadProps }) => {
+  return (
+    <div>
+      <Dragger {...uploadProps}>
+        <p className="ant-upload-drag-icon">
+          <Icon icon="inbox" />
+        </p>
+        <p className="ant-upload-text">Click or drag file to this area to upload</p>
+        <p className="ant-upload-hint">
+          Support for a single or bulk upload. Strictly prohibit from uploading company data or
+          other band files
+        </p>
```

Comment:
I'm guessing you wrote this from the design file? Need to talk with @lavalse since this is confusing enough that if I guess how to fix it, Im not 100% sure it'll convey the correct message.